logback/src/main/java/com/newrelic/logging/logback/NewRelicJsonLayout.java

```diff
@@ -23,6 +23,16 @@
 import static com.newrelic.logging.core.ExceptionUtil.MAX_STACK_SIZE;
 
 public class NewRelicJsonLayout extends LayoutBase<ILoggingEvent> {
+    private Integer maxStackSize;
```

Comment:
Can we make this `final`?

Reply:
> This is great, thanks for the contribution! Is this settable via XML as-is? If not, what would it take?

I'll create a Java project using XML syntax and see what it would take. Should be pretty easy.

Reply:
For sure.